logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/ConversationScope.kt

```diff
@@ -87,7 +90,7 @@ class ConversationScope internal constructor(
     private val persistMessage: PersistMessageUseCase,
     private val updateKeyingMaterialThresholdProvider: UpdateKeyingMaterialThresholdProvider,
     private val selfTeamIdProvider: SelfTeamIdProvider,
-    private val sendConfirmation: SendConfirmationUseCase,
+    private val sendConfirmation: SendConfirmationUseCaseImpl,
```

Comment:
Does it really need to be receiving the implementation here instead of the interface? 🤔

Reply:
OOoh yeah.

I used the refactor tool from the IDE to change the name of the class to adjust constructors, and had this side effect that I forgot to fix 😄 

Nice catch.